walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -58,12 +57,12 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 
 //Home > Search > Select Templates from the drop down > Search for datasync and they should show up. Just make sure you're on the openshift namespace
 
-. Go to the link:{openshift-host}/console/project/{walkthrough-namespace}/overview[project console, window="_blank"] in OpenShift.
-. Click the *Browse Catalog* button.
-. Search the service catalog for 'Data Sync Showcase'.
+. Go to the link:{openshift-host}/topology/ns/{walkthrough-namespace}[project console, window="_blank"] in OpenShift.
+. Click the *From Catalog* button.
+. Search the developer catalog for 'Data Sync Showcase'.
 . Click on the 'Data Sync Showcase' entry to add it to your project.
 . Accept all the defaults of the provisioning wizard and click *Create*.
```

Comment:
@darahayes This was a form on my screen instead of a wizard.

Reply:
```suggestion
. Accept all the defaults of the provisioning form and click *Create*.
```